prowler/providers/aws/services/route53/route53_dangling_ip_subdomain_takeover/route53_dangling_ip_subdomain_takeover.py

```diff
@@ -7,6 +7,14 @@
 from prowler.providers.aws.services.route53.route53_client import route53_client
 
 
+def validate_ip_address(ip_string):
+    try:
+        ip_address(ip_string)
+        return True
+    except ValueError:
+        return False
+
+
```

Comment:
Please move this function to another library or service to reuse it.

Reply:
Done